tests/contenttypes_tests/test_order_with_respect_to.py

```diff
@@ -1,18 +1,18 @@
-from order_with_respect_to.tests import (
-    OrderWithRespectToTests, OrderWithRespectToTests2,
+from order_with_respect_to.base_tests import (
+    BaseOrderWithRespectToTests, BaseOrderWithRespectToTests2,
 )
 
 from .models import Answer, Post, Question
 
 
-class OrderWithRespectToGFKTests(OrderWithRespectToTests):
+class OrderWithRespectToGFKTests(BaseOrderWithRespectToTests):
     Answer = Answer
     Question = Question
 
-del OrderWithRespectToTests
+del BaseOrderWithRespectToTests
```

Comment:
Alternately to this pattern, the base classes could inherit `object` (so the test runner doesn't pick them up) and the subclasses could inherit the base class as a mixin as well as `TestCase`. This is slightly repetitive though, so I couldn't quite decide which is better.